src/npc.cpp

```diff
@@ -104,6 +104,36 @@ npc::npc()
     last_updated = calendar::turn;
 }
 
+standard_npc::standard_npc( const std::string &name, const std::vector<itype_id> &clothing,
+                            int sk_lvl, int s_str, int s_dex, int s_int, int s_per )
+{
+    this->name = name;
+    position = { 0, 0, 0 };
```

Comment:
The value is supposed to cause debugmsgs when an un-positioned NPC does something.
It isn't magic, except for the part where it is outside the allowed range.